siff-dr-router/.gitignore

```diff
@@ -0,0 +1 @@
+router-main
```

Comment:
Why does this folder have a separate gitignore? Is there a reason we don't want the main in git?